### PR TITLE
Add contextual links to demos

### PR DESCRIPTION
**Pull request by Strift**
# Pull Request

## What does this PR do?

- Add links to relevant demos on the screenshots
- Add links to the SaaS demo on those pages: tenant tokens, sorting, tenant tokens, multi-search API
- Add links to the Ecommerce demo on those pages: faceted search, sorting

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

File: reference/api/multi_search.mdx [hunk 1]
```diff
@@ -7,6 +7,10 @@ description: The /multi-search route allows you to perform multiple search queri
 
 The `/multi-search` route allows you to perform multiple search queries on one or more indexes by bundling them into a single HTTP request. Multi-search is also known as federated search.
 
+## Example application
+
+Meilisearch CRM SaaS demo uses multi-search to allow searching through contacts, deals, and companies via a unified search bar. Check out the [SaaS demo](https://saas.meilisearch.com/?utm_campaign=oss&utm_source=docs&utm_medium=multi-search-api) and the [Github repository](https://github.com/meilisearch/saas-demo/).
+
```
Comment (guimachiavelli) [inline reference/api/multi_search.mdx:13]: Could we skip this one? I really think it's a bit out of place to have it in a reference page, where users come to get hard data on how to use the API.

```suggestion
```

**Review by guimachiavelli — CHANGES_REQUESTED** [review]
We are unfortunately required to use the official GitHub spelling. It's clumsy, but 🤷 

Also, as I mention in my comment, I'm not big on the idea of adding new sections to the reference pages. They have a fairly strict structure so users can find info quickly.

File: reference/api/multi_search.mdx (hunk 1 above)
Comment (Strift) [reply reference/api/multi_search.mdx:13]: Do you have another page to recommend to include this link?

It's the only one I found that focuses on multi-search

**Comment by Strift** [comment]
Hello,

For simplicity, I approved your changes removing the demo in reference. 

However, do you have another page to recommend to include this link? It's the only one I found that focuses on multi-search, and the laravel demo is the only that showcase multi search :(

**Comment by guimachiavelli** [comment]
Hmmm…good point. The only other place we mention multi-search is the [Meilisearch overview](https://www.meilisearch.com/docs/learn/what_is_meilisearch/overview). Perhaps we could add it there for the moment? It's not ideal, but it's one of the most visited docs pages.

In the future I'd like to have a simple multi-search guide which would be the ideal place for a demo link, but I really can't give you anything resembling a timeline right now.

**Comment by Strift** [comment]
I'm happy to merge as-is and consider where to include that link in a future PR. 

Feel free to merge whenever possible  🙏

**Review by guimachiavelli — APPROVED** [review]
Sorry, lost sight of this for the past couple of weeks.

Merging!

**Pull request merged**